### ISSUE TITLE
Jaws not recognising links in Chromium browsers

### ISSUE THREAD

**Issue by LJWatson** [FreedomScientific/VFO-standards-support#212]
### Summary
Jaws does not consistently announce the role of anchor elements in Chromium browsers.

1. Go to https://www.nationwide.co.uk/support/contact-us/call-us#tab:Callus
2. Press h to move to the h1 ("Support").
3. Press l to move to the next list (of 6 items).
4. Use the tab key to navigate through the links inside the list.

### Expected result
Jaws should announce the link text and the role of the anchor element. For example "Call us link", "Branch finder link" etc.

### Actual result
Jaws only announces the link text, it does not announce the role of the anchor element. For example "Call us", or "Branch finder".

### Example
Live example on the [Nationwide support page](https://www.nationwide.co.uk/support/contact-us/call-us#tab:Callus)

#### JAWS version and build number
Jaws 2019. 1906.10
 
#### Operating System and version
Windows 10 Pro.

#### Browser and version:
Edge 77, Chrome 75.

**Comment by JAWS-test** [comment]
The reason is that the links were marked with display:table-cell. Chrome thus transmits the element without a role via the accessibility interface. In this respect, it is actually an error by Chrome and not by JAWS. However, JAWS could also evaluate the DOM and determine that these are links.

**Comment by lamador** [comment]
The issue was not reproducible using the latest version of JAWS (2020.2006.12), Chrome (83.0.4103.97) and Edge (83.0.478.45) on Windows 10.
